log10/feedback/feedback_task.py

```diff
@@ -62,3 +77,49 @@ def create_feedback_task(name, task_schema, instruction):
     task_schema = json.loads(task_schema)
     task = FeedbackTask().create(name=name, task_schema=task_schema, instruction=instruction)
     click.echo(f"Use this task_id to add feedback: {task.json()['id']}")
+
+
+@click.command()
+@click.option("--limit", default=25, help="Number of feedback tasks to fetch")
+@click.option("--offset", default=0, help="Offset for the feedback tasks")
+def list_feedback_task(limit, offset):
+    res = FeedbackTask().list(limit=limit, offset=offset)
+    feedback_tasks = res.json()
+
+    data_for_table = []
+
+    for task in feedback_tasks["data"]:
+        data_for_table.append(
+            {
+                "id": task["id"],
+                "created_at": _get_time_diff(task["created_at"]),
+                "name": task["name"],
+                "required": task["json_schema"]["required"],
+                "instruction": task["instruction"],
+            }
+        )
+
+    table = Table(title="Feedback Tasks")
+    table.add_column("ID", style="dim")
+    table.add_column("Created At")
+    table.add_column("Name")
+    table.add_column("Required")
+    table.add_column("Instruction")
+    for item in data_for_table:
+        required = ", ".join(item["required"]) if item["required"] else ""
+        table.add_row(item["id"], item["created_at"], item["name"], required, item["instruction"])
+
+    console = Console()
+    console.print(table)
+
+    # prompt user to put a id to get details info
+    while True:
```

Comment:
Maybe we just have a `log10 feedback_task get <id>` instead? Seems a bit unusual to drop them into a input prompt here.